src/buildCommon.js

```diff
@@ -233,7 +235,7 @@ const makeOrd = function<NODETYPE: "spacing" | "mathord" | "textord">(
     group: ParseNode<NODETYPE>,
     options: Options,
     type: "mathord" | "textord",
-): domTree.symbolNode | domTree.documentFragment {
+): domTree.symbolNode | HtmlDocumentFragment {
```

Comment:
Changing the order, i.e., `HtmlDocumentFragment | domTree.symbolNode` seems to work.

Reply:
This seems to fix other three `$FlowFixMe`s in `src/functions/*.js`.

Reply:
Great! Excellent catch! The flow error could definitely do with some improving.
Updated.